rotkehlchen/icons.py

```diff
@@ -44,8 +44,11 @@ def __init__(self, data_dir: Path, coingecko: Coingecko) -> None:
         self.custom_icons_dir.mkdir(parents=True, exist_ok=True)
         self.failed_asset_ids: set[str] = set()
 
+    def _default_name(self, identifier: str) -> str:
```

Comment:
I am not sure that I understand. Why to create this method?

Reply:
Created it because I needed the name in other places too but as how I rewrote it is not needed anymore. Will undo this change